jni_gen_test/test/jni_gen_test_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:io';
+
+import 'package:jni/jni.dart';
+
+import 'package:jni_gen_test/dev/dart/jni_gen_test.dart';
+import 'package:test/test.dart';
+
+void main() {
+  if (!Platform.isAndroid) {
+    Jni.spawn(helperDir: 'build/jni_libs', classPath: ['java/']);
+  }
+
+  test('static final fields', () {
+    expect(Example.ON, equals(1));
+    expect(Example.OFF, equals(0));
+  });
+
+  test('static & instance fields', () {
+    expect(Example.num, equals(121));
+
+    final aux = Example.aux;
```

Comment:
Generate a comment on the `aux` getter it creates a reference from Dart to Java that must be `delete`d later.